source/_docs/terminus/scripting.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Terminus Manual
+subtitle: Scripting Terminus
+description: Automate your workflow with Terminus.
+terminuspage: true
+type: terminuspage
+layout: terminuspage
+nexturl: terminus/plugins/
+previousurl: terminus/commands/
+permalink: docs/terminus/:basename/
+image: terminus-thumbLarge
+searchboost: 100
+---
+
+While Terminus makes it easy to directly interact with Pantheon from your command line, its real value is in scripting. By adding Terminus to your automated tasks, you can truly make Pantheon a part of your standardized workflow.
+
+Consider the repetitive tasks you perform using the Pantheon Dashboard:
+
+ - Can those tasks be executed by Terminus commands?
+ - Can the values required by the commands be derived programmatically?
+
+If so, you consider how you can turn the task into a script.
+
+## Authentication
+
+Terminus needs to be authenticated in order to execute most commands. Before running any script you must ensure Terminus is authenticated with a [machine token](/docs/terminus/install#machine-token) that has the proper permissions
+
+## Examples
+
+ - The [Pantheon Example Terminus Auto Update Script](https://github.com/pantheon-systems/example-terminus-auto-update-script){.external} demonstrates how you can use Terminus to automate plugin and theme updating for multiple sites.
+
+
+ - The [Example WordPress Composer](https://github.com/pantheon-systems/example-wordpress-composer){.external} repository (used by our [Build Tools](/docs/guides/build-tools/) guide) uses Terminus to [deploy staged changes](https://github.com/pantheon-systems/example-wordpress-composer/blob/46ff34e2b9f421a1c0eae72ade80376e8dd42f31/.circleci/deploy-to-pantheon.sh){.external} to multidev environments.
+
+
+
+## Bash Variables
+
+One of the ways Terminus can be used in script is the generation of environment variables. In the example below, we use the output of `terminus multidev:list` to create an environment variable with all our multidev environments:
```

Comment:
> One of the ways Terminus can be used in script is the generation of environment variables. In the example below, we use the output of terminus multidev:list to create an environment variable with all our multidev environments

"can be used in script" doesn't sound quite right. Perhaps "used in scripts" (plural) or "used in scripting"

Also, it should be just "variables", not "environment variables". Environment variables are variables set by the environment (like CircleCI itself), not by the user.

Reply:
Resolved in f1c1067